walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/rawrequests/VerifiedIdOpenIdJwtRawRequest.kt

```diff
@@ -24,9 +27,31 @@ internal class VerifiedIdOpenIdJwtRawRequest(
             throw MissingRequestStateException("Request State is missing in presentation request")
         if (rawRequest.content.redirectUrl.isEmpty())
             throw MissingCallbackUrlException("Callback url is missing in presentation request")
+        val requirementsList = rawRequest.getPresentationDefinitions().map { it.toRequirement() }
+        val requirement = if (requirementsList.size == 1) {
+            requirementsList.first()
+        } else {
+            var groupRequirement = true
```

Comment:
nit: This is to indicate if requirement is required. naming conveys if it is a group requirement.